src/store/plugins/errorHandler.js

```diff
@@ -0,0 +1,40 @@
+export default ({ dispatch }) => {
+  const errors = [];
+  let invoke = true;
+  const handlerBody = async (error) => {
+    if (error) errors.unshift(error);
+    if (invoke && errors.length !== 0) {
+      invoke = false;
+      await dispatch('modals/open', {
+        name: 'error',
+        error: errors.shift(),
+      });
+      invoke = true;
+      handlerBody();
+    }
+  };
+
+  const handlerBodyLoop = async (error) => {
+    if (error) errors.unshift(error);
+
+    for (let i = 0; i < errors.length; i++) {
+      if (invoke && errors.length !== 0) {
+        invoke = false;
+        await dispatch('modals/open', {
+          name: 'error',
+          error: errors.shift(),
+        });
+        invoke = true;
+      }
+    }
+  };
```

Comment:
@davidyuk can you have a look at handlerBodyLoop function?

Reply:
I would do it this way:
```suggestion
  const handlerBodyLoop = async (error) => {
    errors.unshift(error);
    if (!invoke) return;
    invoke = false;
    while (errors.length) {
      // eslint-disable-next-line no-await-in-loop
      await dispatch('modals/open', {
        name: 'error',
        error: errors.shift(),
      });
    }
    invoke = true;
  };
```